frontend/src/components/organisms/NewForm/NewForm.js

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from 'react';
+import {
+  Form, Select, Divider, Row, Col, Space,
+} from 'antd';
+import ctgov from '../../../apis/ctgov';
+import TextInputField from '../../atoms/TextInputField/TextInputField';
+import SelectField from '../../atoms/SelectField/SelectField';
+import Button from '../../atoms/buttons/Button/Button';
+import BasicSearchAccordion from '../../molecules/BasicSearchAccordion/BasicSearchAccordion';
+
+import './NewForm.css';
+
+import {
+  recruitment, access, countries,
+} from '../../../variables/TopLevelSearchData';
+
+class NewForm extends Component {
+  constructor(props) {
+    super(props);
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleCountryChange = this.handleCountryChange.bind(this);
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleClear = this.handleClear.bind(this);
+    this.state = {
+      condition: '',
+      target: '',
+      country: '',
+      otherTerms: '',
+      searchResults: '',
+    };
+  }
+
+  handleInputChange(e) {
+    const { target } = e;
+    const value = target.inputType === 'checkbox' ? target.checked : target.value;
+    const { name } = target;
+
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  handleCountryChange(e) {
+    this.setState({
+      country: e,
+    });
+  }
+
+  async handleSearch() {
+    const payload = {
+      status: '',
+      condition: this.state.condition,
+      other_terms: this.state.otherTerms,
+      country: this.state.country,
+      intervention: this.state.intervention,
+      target: this.state.target,
+      nct_id: this.state.nct_id,
+      eligibility_criteria: '',
+      first: '',
+      last: '',
+    };
+    try {
+      const response = await ctgov.post('basic_search', payload);
+      this.setState({
+        searchResults: response.data,
+      });
+    } catch (err) {
+      console.log(err);
```

Comment:
Maybe another way to do these would be with logging library if you want to keep them in https://www.loggly.com/blog/best-practices-for-client-side-logging-and-error-handling-in-react/